MapboxCoreNavigation/MBRouteController.m

```diff
@@ -17,5 +17,8 @@
 NSString *const MBRouteControllerDidReroute                     = @"RouteControllerDidReroute";
 NSString *const MBRouteControllerDidFailToReroute               = @"RouteControllerDidFailToReroute";
 NSString *const MBRouteControllerDidFindFasterRouteKey          = @"RouteControllerDidFindFasterRoute";
+
+/**
+ Domain for errors created within this framework.
+ */
```

Comment:
Merge this comment with the one in the header. A public Objective-C symbol should be documented only on the declaration (in the header, in this case), not on the definition.